torch/csrc/autograd/function.cpp

```diff
@@ -14,6 +14,14 @@
 #include <utility>
 #include <vector>
 
+namespace torch { namespace jit {
+
+// Sigh, see https://stackoverflow.com/questions/8016780/undefined-reference-to-static-constexpr-char
+constexpr Symbol CppOp::Kind;
+constexpr Symbol PythonOp::Kind;
```

Comment:
Hmm does it really fix the problem? It shouldn’t matter in which cpp file they are defined I think